packages/dnb-eufemia/src/components/upload/__tests__/UploadFileListCell.test.tsx

```diff
@@ -322,6 +322,21 @@ describe('UploadFileListCell', () => {
       ).toBeInTheDocument()
     })
 
+    it('renders the delete button as disabled when loading state', () => {
+      render(
+        <UploadFileListCell
+          {...defaultProps}
+          uploadFile={{
+            file: createMockFile('file.png', 100, 'image/png'),
+            isLoading: true,
+          }}
+        />
+      )
+      const element = screen.getByRole('button')
+
+      expect(element).toHaveAttribute('disabled')
```

Comment:
```suggestion
      expect(element).toBeDisabled()
```

I'm fine with both. I don't think `toBeDisabled` makes any big of a difference.